database/migrations/2024_09_09_181717_create_cases_started_table.php

```diff
@@ -14,7 +13,7 @@ public function up(): void
         Schema::create('cases_started', function (Blueprint $table) {
             $table->id();
             $table->unsignedInteger('case_number')->unique();
-            $table->unsignedInteger('user_id');
+            $table->unsignedInteger('user_id')->nullable();
```

Comment:
Why the `user_id` was changed to `null`?
```suggestion
            $table->unsignedInteger('user_id')->nullable();
```

Reply:
because  user_id is null for  some process like (Expense Report)